IOHexperimenter/Cpp.md

```diff
@@ -29,35 +29,39 @@ If you want to change build options, check the output of `cmake -L` or use `cmak
 
 
 <a name="using-individual-problems"></a>
-## Testing Problems 
+## Testing on Problems 
 
-Users can test algorithms on specific __Problem__ classes. Twenty-five pseudo-Boolean optimization problems, twenty-four bbob problems, and two classes of W-Model problems are available in __IOHexperimenter__.
+Users can test algorithms on built-in `problem` classes. Twenty-five pseudo-Boolean optimization problems ([PBO](https://www.sciencedirect.com/science/article/pii/S1568494619308099)), twenty-four continuous optimization problems ([bbob](https://coco.gforge.inria.fr/downloads/download16.00/bbobdocfunctions.pdf)), and two classes of [W-Model](https://dl.acm.org/doi/pdf/10.1145/3205651.3208240) problems are available in __IOHexperimenter__.
```

Comment:
Maybe we should mention explicitly that all problems share this common problem-class, and that several types of problems are already provided. Then it might be clearer that the items you describe in this section are the ones which the algorithm needs to interact with.